unglueit.py

```diff
@@ -0,0 +1,43 @@
+from nose.tools import set_trace
```

Comment:
I didn't write a test for this code because I'm holding out hope that unglue.it will change the way they present these links (e.g. by using indirectAcquisition) and let me get rid of this code.